src/main/rx/cc2500_frsky_common.h

```diff
@@ -27,6 +27,7 @@ typedef struct rxFrSkySpiConfig_s {
     int8_t  bindOffset;
     uint8_t bindHopData[50];
     uint8_t rxNum;
+    bool useExternalAdc;
```

Comment:
Is bool intended? I would use uint8_t for PGs

Reply:
Probably a better approach. It was copy/paste from `autoBind` above.